### PR TITLE
Add the test of adding node in VMWare since it's now fixed

### PR DESCRIPTION
**Pull request by yitzhak12**
The BZ https://bugzilla.redhat.com/show_bug.cgi?id=1844521 is closed, and Vijay fixed the functionality of adding a new worker node in VMWare, so we can remove the skip line and add the test of adding new worker nodes on VMWare.

### CODE REVIEW

**Comment by yitzhak12** [comment]
fix https://github.com/red-hat-storage/ocs-ci/issues/2403

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py [hunk 1]
```diff
@@ -54,17 +54,17 @@ def test_add_node(self):
...
             )
-            # Issue to remove skip code https://github.com/red-hat-storage/ocs-ci/issues/2403
-            # logger.info(f'The worker nodes number before expansion {len(node.get_worker_nodes())}')
-            # if config.ENV_DATA.get('rhel_user'):
-            #     pytest.skip("Skipping add RHEL node, code unavailable")
-            # node_type = constants.RHCOS
-            # assert add_new_node_and_label_upi(node_type, new_nodes), "Add node failed"
-            # logger.info(f'The worker nodes number after expansion {len(node.get_worker_nodes())}')
+            if config.ENV_DATA.get("rhel_user"):
+                pytest.skip("Skipping add RHEL node, code unavailable")
```
Comment (sidhant-agrawal) [inline tests/manage/z_cluster/cluster_expansion/test_node_expansion.py:61]: I think same `add_new_node_and_label_upi()` can be used here, only `node_type` will be different for RHEL. 
Isn't it?

**Comment by vavuthu** [comment]
@yitzhak12 Please hold off this PR ,  I am going to raise the new PR for add functionality in vSphere using terraform

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py (hunk 1 above)
Comment (yitzhak12) [reply tests/manage/z_cluster/cluster_expansion/test_node_expansion.py:61]: Yes, I think you right.

**Comment by yitzhak12** [comment]
Okay @vavuthu, np. Let me know when you finish

**Review by fbalak — APPROVED** [review]

**Review by sidhant-agrawal — APPROVED** [review]

**Review by shreekarSS — APPROVED** [review]

**Comment by vasukulkarni** [comment]
needs verification label, can you run this test in jenkins for this PR?

**Review by ebenahar — APPROVED** [review]

**Pull request merged**